pkg/api/types.go

```diff
@@ -1205,6 +1221,13 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// Local represents a directory with specific topology contraints
+type LocalVolumeSource struct {
+	// The full path to the volume on the node
+	// For alpha, this path must be a file-based volume
+	Path string
```

Comment:
GCE PD is **always** a block device and Kubernetes makes a fs on it and mounts it. With LocalVolumeSource, now it's a filesystem (or even a shared directory) formatted and mounted outside of Kubernetes and we should have a clean way how to extend it to block device formatted and mounted by Kubernetes.